ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -223,6 +230,29 @@ public class BottomSheetController: UIViewController, Shadowable, TokenizedContr
         }
     }
 
+    /// Setting this property  will result in the sheet trying to be as close to this width as possible.
+    /// If the declared width is too large it will roll back to the maximum width
+    @objc open var preferredWidth: CGFloat = 0 {
+        didSet {
+            guard preferredWidth != oldValue && isViewLoaded else {
```

Comment:
Random small thing - `guard` statements are essentially a negation, so "complex" guard statements with && or || become kinda difficult for simpletons like me to read and parse because I have to do boolean logic in my head. Would recommend either using `,` syntax to join conditions or just splitting out the complexity to a separate bool. In this case, possible recommendation which avoids the need for early returns as well:

```
let shouldInvalidateLayout = (preferredWidth != oldValue) && isViewLoaded
if shouldInvalidateLayout {
    view.setNeedsLayout()
}
 ```